packages/core/src/configuration/Configuration.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { Configuration, container } from '.';
+import { DefaultCredentialProvider } from './DefaultCredentialProvider';
+import { env } from 'process';
+
+describe('configuration', () => {
+  const testApi = 'test/api';
+  const testBus = 'test-bus';
+  const testToken = 'test-token';
+  let configuration: Configuration;
+
+  beforeAll(() => {
+    const credentialProvider = container.resolve(DefaultCredentialProvider);
```

Comment:
It can't be considered as a unit test since you use other dependencies alongside SUT. Use mocks.